sdks/python/apache_beam/testing/util_test.py

```diff
@@ -38,6 +38,14 @@ def test_assert_that_passes(self):
     with TestPipeline() as p:
       assert_that(p | Create([1, 2, 3]), equal_to([1, 2, 3]))
 
+  def test_assert_that_passes_order_does_not_matter(self):
+    with TestPipeline() as p:
```

Comment:
Are these changes (to util_test.py) intended ? Looks like a merge issue.

Reply:
Everything else looks good. Thanks.

Reply:
not sure how to get that removed